docs/blog/newsletter-may-2020.rst

```diff
@@ -0,0 +1,91 @@
+* Share what you understand about the topic, and ask the SMEs for their input. Phrase it as a “yes/no” question (e.g., “This is how I think X works. Is this right?”). If you’re correct, they’ll agree. If you’re wrong, they’ll correct you. This approach is known as Cunningham’s Law: "the best way to get the right answer on the internet is not to ask a question; it's to post the wrong answer."
+
+-------------------------------------
+Tips for creating quality screenshots
+-------------------------------------
+
+Love them - because they make it easier for readers to find things in the UI? Loathe them - because of the maintenance burden? Either way, many of us work a little or a lot with screenshots. And if you're going to have them, it's worth doing them well; so here are a few tips for taking good screenshots:
+
+*   As long as the area of focus is up to date in a screenshot or GIF, updates to other areas shown aren't as urgent. You can wait until the UI changes are highly noticeable or cause confusion.
+*   Consider using a disclaimer to explain possibly confusing differences between your screenshots and your user's screen (for example, due to versioning or feature enablement).
+*   `Simplified UI Screenshots <https://www.techsmith.com/blog/simplified-user-interface/>`__ (where you remove or cover UI elements that you aren't referring to) can require fewer updates and save on translation costs.
+*   Some community members had tried using line drawings instead of screenshots, to be less affected by small UI changes. But they can be even more difficult to maintain than screenshots, so carefully consider how useful one is before creating it.
+*   If you've got a large number of screenshots to maintain, look into automating your screen captures with Selenium.
+
+Screen capture tool recommendations:
+
+*   `Snagit from Techsmith <https://www.techsmith.com/screen-capture.html>`__ (Paid)
+*   `Skitch from Evernote <https://evernote.com/products/skitch>`__ (Free, MacOS & iOS only)
+*   `Greenshot <https://getgreenshot.org/>`__ (Free, OSS)
+*   `Screenpresso <https://www.screenpresso.com/>`__ (Free)
+*   `LightShot <https://app.prntscr.com/en/index.html>`__ (Free)
+*   `Adobe Illustrator <https://www.adobe.com/products/illustrator.html>`__ (Paid)
+*   `Adobe Photoshop <https://www.adobe.com/products/photoshop.html>`__ (Paid)
+
+Join us in `#doctools <https://app.slack.com/client/T0299N2DL/C4EPE8332>`__ if you have any other tool recommendations!
+
+-----------------------------------------
+Choosing the right level to write a topic
+-----------------------------------------
+
+This month, the `#learn-tech-writing <https://app.slack.com/client/T0299N2DL/C7YJR1N02>`__ book club had some interesing discussion based on *Every Page is Page One’s*, on the topic of topics: how do you decide what should go in one? And what's the right level to pitch it at?
```

Comment:
I'm confused about the "'s" in the _Every Page  is Page One_ title...is there something meant to come after the title?

Reply:
just a typo... fixed!